src/transport/SecureSession.h

```diff
@@ -75,7 +75,8 @@ class SecureSession : public Session
                   FabricIndex fabric, const ReliableMessageProtocolConfig & config) :
         mSecureSessionType(secureSessionType),
         mPeerNodeId(peerNodeId), mPeerCATs(peerCATs), mLocalSessionId(localSessionId), mPeerSessionId(peerSessionId),
-        mLastActivityTime(System::SystemClock().GetMonotonicTimestamp()), mMRPConfig(config)
+        mLastActivityTime(System::SystemClock().GetMonotonicTimestamp()),
+        mLastPeerActivityTime(System::SystemClock().GetMonotonicTimestamp()), mMRPConfig(config)
```

Comment:
Initialize to zero just like for UnauthenticatedSession?

Reply:
Was thinking about this, and it makes sense to initialize secure sessions to `now()` as there has been proof of peer activity when the session was created.  Agree?